files/disable_inactive_freeipa_users.sh

```diff
@@ -69,59 +69,76 @@ else
       sed --quiet 's/^\s*User login:\s*//p'
   )
 
+  # Grab the domain. Note that # --quiet means no printing unless the
+  # p command is used.
```

Comment:
Matches how this is done in other parts of this script.
```suggestion
  # Grab the domain.
  #
  # --quiet means no printing unless the p command is used
```